app/helpers/template_helpers.py

```diff
@@ -1,304 +1,371 @@
+    copyright_declaration: LazyString = lazy_gettext(
+        "Crown copyright and database rights 2021 NIMA MOU577.501."
+    )
+    copyright_text: LazyString = lazy_gettext(
+        "Use of address data is subject to the terms and conditions."
+    )
+    footer_links: Iterable[MutableMapping] = field(
+        default_factory=lambda: [
+            Link(
+                lazy_gettext("Help"),
+                f"{CENSUS_NIR_BASE_URL}/help/help-with-the-questions/online-questions-help/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Contact us"), f"{CENSUS_NIR_BASE_URL}/contact-us/"
+            ).__dict__,
+        ],
+        compare=False,
+        hash=False,
+    )
+    footer_legal_links: Iterable[Mapping] = field(
+        default_factory=lambda: [
+            Link(lazy_gettext("Cookies"), f"{CENSUS_NIR_BASE_URL}/cookies/").__dict__,
+            Link(
+                lazy_gettext("Accessibility statement"),
+                f"{CENSUS_NIR_BASE_URL}/accessibility-statement/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Privacy and data protection"),
+                f"{CENSUS_NIR_BASE_URL}/privacy-and-data-protection/",
+            ).__dict__,
+            Link(
+                lazy_gettext("Terms and conditions"),
+                f"{CENSUS_NIR_BASE_URL}/terms-and-conditions/",
+            ).__dict__,
+        ],
+        compare=False,
+        hash=False,
+    )
+    powered_by_logo: str = "nisra-logo-black-en"
+    powered_by_logo_alt: str = "NISRA - Northern Ireland Statistics and Research Agency"
+    account_service_url = CENSUS_NIR_BASE_URL
+    data_layer: Iterable[Mapping] = field(
+        default_factory=lambda: [{"nisra": True}], compare=False
+    )
 
-def get_data_layer(schema_theme: str) -> List:
-    if schema_theme == "census-nisra":
-        return [{"nisra": True}]
 
-    if schema_theme == "census":
-        return [{"nisra": False}]
+@lru_cache
+def survey_config_mapping(theme: str, language: str) -> SurveyConfig:
+    return {
+        "business": SurveyConfig,
+        "health": SurveyConfig,
+        "social": SurveyConfig,
+        "census": (
+            CymraegCensusSurveyConfig if language == "cy" else CensusSurveyConfig
+        ),
+        "default": SurveyConfig,
+        "census-nisra": CensusNISRASurveyConfig,
+    }[theme]()
+
+
+def get_survey_config(
+    theme: Optional[str] = None, language: Optional[str] = None
+) -> SurveyConfig:
+    if not language:
+        language = get_locale().language
+    if not theme:
+        theme = cookie_session.get("theme", current_app.config["SURVEY_TYPE"])
+    survey_config = survey_config_mapping(theme, language)
+
+    return survey_config
+
+
+def render_template(template: str, **kwargs: Union[str, Mapping]) -> str:
+    # The fallback to assigning SURVEY_TYPE to theme is only being added until
+    # business feedback on the differentiation between theme and SURVEY_TYPE.
```

Comment:
This comment needs to move into the `get_survey_config` function.

Reply:
Done